ulib/axlibc/c/signal.c

```diff
@@ -12,6 +12,8 @@
 #include <stddef.h>
 #include <stdio.h>
 
+extern int sigaction_inner(int, const struct sigaction *, struct sigaction *);
```

Comment:
This `extern` is not needed. By the way, it is okay to move `sigaction_helper` into `sigaction_inner`.

Reply:
There will be a warning without `extern`, since `sigaction_inner` is defined in another file.